cakefuzzer/instrumentation/info_retriever.py

```diff
@@ -51,6 +51,13 @@ async def _call_app_info(
             exit()
         return output
 
+    @property
+    async def paths(self) -> List[str]:
```

Comment:
Is it a list of strings or a dict? I thought it was meant to be dict now, right ?